### ISSUE TITLE
[Accessibility][Tooltiphost]: NVDA+Firefox tooltiphost creates repeated screen reader experience

### ISSUE THREAD

**Issue by yaananth** [OfficeDev/office-ui-fabric-react#1301]
### Bug Report

- __Package version(s)__: latest

#### Describe the issue:
I repro'd this on firefox + nvda (part of industry standards)
http://codepen.io/yashanantha/pen/xqPvEV (with tooltiphost)
 This is what screen reader reads out:
Note: These are the columns that have tooltiphost - 1,2,3,6
```
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
Fabric DetailsList with tooltiphost - Mozilla Firefox
unknown
Fabric DetailsList with tooltiphost - Mozilla Firefox
Fabric DetailsList with tooltiphost  document
http://s.codepen.io/boomerang/iFrameKey-0f929676-97a1-20d0-d34f-f080b1ebc099/index.html  document
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
Item a2
Item a2  button
Fabric DetailsList with tooltiphost - Mozilla Firefox
unknown
Fabric DetailsList with tooltiphost - Mozilla Firefox
Fabric DetailsList with tooltiphost  document
http://s.codepen.io/boomerang/iFrameKey-0f929676-97a1-20d0-d34f-f080b1ebc099/index.html  document
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
Item a3
Item a3  button
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
hello there4!
hello there4!  button
hello there5!
hello there5!  button
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
hello there6!
hello there6!  link  visited
Fabric DetailsList with tooltiphost - Mozilla Firefox
unknown
Fabric DetailsList with tooltiphost - Mozilla Firefox
Fabric DetailsList with tooltiphost  document
http://s.codepen.io/boomerang/iFrameKey-0f929676-97a1-20d0-d34f-f080b1ebc099/index.html  document
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
hello there7!
hello there7!  link  visited
hello there8!
hello there8!  link  visited
hello there9!
hello there9!  link  visited
```
As you can see for columns 1,2,3 reader reads the whole table again.
As it comes to 3 (tooltiphost) => 4 (no tooltiphost) it reads again, but 5 (no tooltiphost) is clean.
Same goes with 6=>7 transition, and 8,9 are good

http://codepen.io/yashanantha/pen/xqPvYR (with out tolltiphost)
It doesn't have column 9, and one duplicate reading but that's not the point 😀
 This is what screen reader reads out:
```
table  editable
list
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
Item a1 Item a2 Item a3 hello there4! hello there5! hello there6! hello there7! hello there8!  row  1 of 1
Item a2
Item a2  button
Item a3
Item a3  button
hello there4!
hello there4!  button
hello there5!
hello there5!  button
hello there6!
hello there6!  link  visited
hello there7!
hello there7!  link  visited
hello there8!
hello there8!  link  visited
```

I think tooltiphost adds and removes 'ms-layer' everytime. So screen reader thinks DOM has changed and reads the whole row again? 
In any case, we should discuss about it and fix it...

**Comment by sandeepchads** [comment]
hi @dzearing  - I see this bug is tagged as needs description. Please let me know if you need any additional information regarding this issue.
